respa/settings.py

```diff
@@ -436,3 +438,13 @@ def get_git_revision_hash():
 #
 if DATABASES['default']['ENGINE'] != 'django.contrib.gis.db.backends.postgis':
     raise ImproperlyConfigured("Only postgis database backend is supported")
+
+# CKEditor config
+CKEDITOR_UPLOAD_PATH = "uploads/"
```

Comment:
Should this be joined to `BASE_DIR` i.e. `CKEDITOR_UPLOAD_PATH = os.path.join(BASE_DIR, "uploads/")` ? Maybe also an environment variable in case we need to move to e.g. a shared volume in production.

Reply:
Yeah, definitely, good point. I'll make the change.

Reply:
Actually, no, I don't think this should be joined with `BASE_DIR`. CKEditor will join the configured path with `MEDIA_ROOT` and the urls will be created against `MEDIA_URL`. So it should be OK as it is.

https://django-ckeditor.readthedocs.io/en/latest/#required-for-using-widget-with-file-upload